apps/web/src/components/Player/PlayerMenuBar/PlayerMenuBar.tsx

```diff
@@ -63,12 +65,21 @@ export default function PlayerMenuBar({ roomName, disconnect }: { roomName?: str
   }, [isHandRaised, appState.participantName, appState.eventName, enqueueSnackbar]);
 
   const toggleParticipantWindow = () => {
+    setIsChatWindowOpen(false);
     appDispatch({
       type: 'set-is-participant-window-open',
       isParticipantWindowOpen: !appState.isParticipantWindowOpen,
     });
   };
 
+  const toggleChatWindow = () => {
+    setIsChatWindowOpen(!isChatWindowOpen);
+    appDispatch({
+      type: 'set-is-participant-window-open',
+      isParticipantWindowOpen: false,
+    });
+  };
+
```

Comment:
This is kinda funny - just that these two pieces of state that control two sidebars live in two different places. Not sure it's worth fixing now though.